src/deploy/table_event_policy.sql

```diff
@@ -13,26 +14,31 @@ GRANT INSERT, UPDATE, DELETE ON TABLE maevsi.event TO maevsi_account;
 
 ALTER TABLE maevsi.event ENABLE ROW LEVEL SECURITY;
 
--- Only display events that are public and not full.
+-- Only display events that are public and not full and not organized by a blocked account
 -- Only display events that are organized by oneself.
--- Only display events to which oneself is invited.
+-- Only display events to which oneself is invited, but not by an invitation authored by a blocked account
```

Comment:
```suggestion
-- Only display events to which oneself is invited, but not by an invitation authored by a blocked account.
```

Let's not start changing the format of comments (same for the line above).